client/js/widgets/scoreboard.js

```diff
@@ -0,0 +1,297 @@
+import { $, $a, removeFromDOM, asArray, escapeID } from '../domhelpers.js';
+import { Widget } from './widget.js';
+
+class Scoreboard extends Widget {
+  constructor(object, surface) {
+    super(object, surface);
+
+    this.addDefaults({
+      movable: true,
+      width: 300,
+      height: 200,
+      layer: -1,
+      typeClasses: 'widget scoreboard',
+      usePlayerColors: true,
+      playersInColumns: true,
+      rounds: null,
+      sortField: 'index',
+      sortAscending: true,
+      scoreProperty: 'score',
+      includeAllSeats: false,
+      showTotals: true,
+      seats: null,
+      showAllRounds: false,
+      roundLabel: 'Round',
+      currentRound: null
+    });
+  }
+
+  applyDeltaToDOM(delta) {
+    if(delta) // Don't call super unless this is a real delta to the scoreboard rather than from a seat
+      super.applyDeltaToDOM(delta);
+    this.tableCreate(this.getIncludedSeats())
+  }
+
```

Comment:
I think there is room/need for a click function. I imagine it similar to the pile click that is some sort of input but just to confirm the player wants to sum another round

Reply:
What is the use case, and how is it different from the scoreboard automatically updating when one or more scores in its seats changes?

Reply:
It would be interesting to have a (maybe optional) click function that lets you edit the scoreboard manually. Maybe even on a per round basis.

That way you could just add a scoreboard without having to write automation.

Reply:
Interesting. Probably not that hard (he said, ignoring all the edge cases and other complexity). Of course, you have to keep the score arrays in the seats up to date, unless you want to say that each scoreboard is either totally manual or totally automated.

Reply:
yeah agree that it should go to the seats

Reply:
@ArnoldSmith86 I'm definitely going to need some help getting the table editability routine up and running. I have something that mostly works, but the issue I'm having is getting clicks in other parts of the room to work correctly. The behavior I'm looking for is: Enter accepts an edit, ESC or a click anywhere else aborts the edit (and a click doesn't do whatever the click would normally do). I'm getting messed up with too many click handlers. Can you help, or can someone else?

Reply:
To be implemented later/some other way.